src/controllers/commands/CommandController.cpp

```diff
@@ -3207,6 +3209,28 @@ void CommandController::initialize(Settings &, Paths &paths)
 
         return "";
     });
+
+    this->registerCommand(
+        "/force-image-gc",
+        [](const QStringList & /*words*/, auto /*channel*/) -> QString {
+            postToThread([]() {
```

Comment:
It is extremely unlikely that this runs outside the gui-thread (e.g. in `execCommand` there's a call to `getApp`), so this should use `runInGuiThread`.